### ISSUE TITLE
inner columns shorter than outer columns

### ISSUE THREAD

**Issue by markamber** [microweber/microweber#340]
Why when I make a column layout are the inner columns shorter than the outer columns? it looks strange.

**Comment by peter-mw** [comment]
can you post a screenshot?

**Comment by markamber** [comment]
Very very easy to reproduce. Just make some columns. The outer ones have less padding than the inner ones. 

-- 
Mark Amber
(815) 713 - 8116

On February 16, 2016 at 2:44:26 PM, Peter Ivanov (notifications@github.com) wrote:

can you post a screenshot?

**Comment by Thomaxo** [comment]
Haven't tried to reproduce this but i think this doesn't have anything to do with the columns themselves but with the (row-)container. The (row-)container also has a padding, so when the padding of the columns and row is set to 15px you'll always have 30px in total, even on the outer sides.

**Comment by markamber** [comment]
Indeed, that is the culprit. default.css:257 RE the first child padding-left:0; I wonder what changing that to 10 will break elsewhere…

-- 
Mark Amber
(815) 713 - 8116

On February 17, 2016 at 5:10:54 PM, Thomas Vansteenkiste (notifications@github.com) wrote:

Haven't tried to reproduce this but i think this doesn't have anything to do with the columns themselves but with the (row-)container. The (row-)container also has a padding, so when the padding of the columns and row is set to 15px you'll always have 30px in total, even on the outer sides.

**Comment by Thomaxo** [comment]
I guess it relies on the container of the columns to have a padding of 10, so just put padding there?
(mw-row probably)
